internal/kldtx/txnprocessor_test.go

```diff
@@ -630,7 +630,7 @@ func TestOnSendTransactionMessageBadMsg(t *testing.T) {
 		"  \"headers\":{\"type\": \"SendTransaction\"}," +
 		"  \"from\":\"0x83dBC8e329b38cBA0Fc4ed99b1Ce9c2a390ABdC1\"," +
 		"  \"nonce\":\"123\"," +
-		"  \"value\":\"abc\"," +
+		"  \"value\":\"123.456\"," +
```

Comment:
Guessing this is a go version behavior change: using `abc` for nonce throws an error in parsing the json in the test file: "json: invalid number literal, trying to unmarshal "\"abc\"" into Number" instead of throwing an error in txn processing logic